src/components/profile/Profile.js

```diff
@@ -32,9 +32,11 @@ class Profile extends Component {
             loader: false
          }))
       }, 1000)
+
```

Comment:
These new lines are required?

Reply:
I used to console log the props, now it is no more used